WBooks/Library/Model/Book.swift

```diff
@@ -7,8 +7,30 @@
 
 import Foundation
 
-struct Book: Codable {
-    var title: String?
-    var autor: String?
-    var image: String?
+public struct Book: Codable {
```

Comment:
Why we do it public?

Reply:
@NickMano  I need to use that structure in other classes, but analyzing could I leave it internal?

Reply:
Yes, we shouldn't have problems :)

Reply:
Done